CONTRIBUTING.md

```diff
@@ -31,10 +31,13 @@ Please be aware of the following things when filing bug reports.
 
 If the bug report is missing this information then it'll take us longer to fix the issue. We will probably ask for clarification, and barring that if no response was given then the issue will be closed.
 
+## Creating a Pull Request
 
-## Submitting a Pull Request
+Submitting a pull request is fairly simple, just make sure it focuses on a single aspect and doesn't manage to have scope creep and it's probably good to go.
```

Comment:
This feels weird, you've changed Submitting -> Creating above, but are then using Submitting again here? It would be better to stick to one or the other.

Reply:
https://github.com/DisnakeDev/disnake/pull/601/commits/53893b007b09347b459d13c295552bd7fc6d37e5